css/custom.css

```diff
@@ -48,6 +57,36 @@ main .feed-icons {
     }
 }
 
+.views-header {
+    padding: 0 0 15px 15px;
+    color: rgba(80, 90, 90, 255);
+}
+
+/* These are the background colors behind the filters above content listings */
+#main .views-element-container .views-exposed-form {
+    padding: 15px;
+    background-color: rgba(0, 0, 0, 0.063);
+    -moz-border-radius: 8px;
+    -webkit-border-radius: 8px;
+    border-radius: 8px; /* future proofing */
+    -khtml-border-radius: 8px; /* for old Konqueror browsers */
+}
+
+#main .views-element-container #edit-header {
+    margin-top: 15px;
+    padding: 15px 0;
+    background-color: rgba(0, 0, 0, 0.063);
+    -moz-border-radius: 8px;
+    -webkit-border-radius: 8px;
+    border-radius: 8px; /* future proofing */
+    -khtml-border-radius: 8px; /* for old Konqueror browsers */
+}
+
+/* This is the "Apply to selected items" button in the filter area above content listings */
+#edit-submit--2 {
```

Comment:
Is this selector "user changes" aware? Any way you can target this without relaying on Drupal's weird-o auto naming? Just a question, not a change request